tools/bin/ecflow/processes/main/post_processing/percentiles.ecf

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+%include <qsub.h>
```

Comment:
Isn't this `qsub_build.h` because we're transferring files? Or do we only update that in the operational system?

Reply:
I figured we'd only update this in the operational system, since this is a workaround that is specific to Hyak.